### PR TITLE
Add etcd into prebuild

### PR DESCRIPTION
**Pull request by LemonHX**
due to etcd is the bedrock of cloud

### CODE REVIEW

File: prebuilts/etcd.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+port = 2380
+type = "TCP"
+
+[[volumes]]
+id = "data1"
+dir = "/var/lib/etcd"
+
+[[volumes]]
+id = "data2"
+dir = "/var/etcd"
```
Comment (STRRL) [inline prebuilts/etcd.toml:25]: etcd uses `/var/lib/etcd` as default data storage location, it seems we do not need this volume

File: prebuilts/etcd.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[[ports]]
+id = "client_requests"
+port = 2379
+type = "TCP"
+
+[[ports]]
+id = "peer_communication"
+port = 2380
+type = "TCP"
```
Comment (STRRL) [inline prebuilts/etcd.toml:17]: if we run single instance etcd, we do not need to export this port.

File: prebuilts/etcd.toml (hunk 1 above)
Comment (LemonHX) [reply prebuilts/etcd.toml:25]: （found in docs

File: prebuilts/etcd.toml (hunk 2 above)
Comment (LemonHX) [reply prebuilts/etcd.toml:17]: (but this is prebuilt perhaps it uses this port

File: prebuilts/etcd.toml (hunk 1 above)
Comment (LemonHX) [reply prebuilts/etcd.toml]: removed

File: prebuilts/etcd.toml (hunk 2 above)
Comment (STRRL) [reply prebuilts/etcd.toml:17]: @yuaanlin told me that zeabur could only export only one port for each container, so I am not sure `2379` and `2380` could be both exported.

Comment (yuaanlin) [reply prebuilts/etcd.toml:17]: If a prebuilt service has two tcp port, both can access from other services in the same environment. But if you want to connect it via public network (port forwarding), only one port can access.

**Review by yuaanlin — APPROVED** [review]

**Pull request merged**